src/components/SideNavBar/SideNavBarItemGroup.jsx

```diff
@@ -1,22 +1,40 @@
-import React, { Component } from 'react';
-import PropTypes from 'prop-types';
+// @flow
+import React, { Component, type Node } from 'react';
 import { find } from 'lodash';
 import SideNavBarItem from './SideNavBarItem';
+import { type Item } from './SideNavBar';
 import style from './style.scss';
 
-class SideNavBarItemGroup extends Component {
+
+type Props = {
+  id: string,
+  icon: string,
+  iconSize: number,
+  iconStyle?: Object,
+  label: Node | string,
```

Comment:
https://flow.org/en/docs/react/types/#toc-react-node

Node could be a string already so this is redundant